app/views/user_invite_requests/new.html.erb

```diff
@@ -12,11 +12,12 @@
 
 <!--main content-->
 <%= form_for(@user_invite_request) do |f| %>
+<p class="required notice"><%= ts("* Required information") %></p>
   <%= error_messages_for @user_invite_request %>
   <dl>
-    <dt class="required"><%= f.label ts("How many invitations would you like? (max %{max})", :max => ArchiveConfig.MAX_USER_INVITE_REQUEST.to_s) %></dt>
+    <dt class="required"><%= f.label ts("How many invitations would you like? (max %{max})*", :max => ArchiveConfig.MAX_USER_INVITE_REQUEST.to_s) %></dt>
```

Comment:
When we put asterisks on labels, we like to keep them outside the `ts()` to make sure they can't get lost during the translation process, like so: `<%= f.label ts("How many invitations would you like? (max %{max})", :max => ArchiveConfig.MAX_USER_INVITE_REQUEST.to_s) + "*" %>`

Reply:
Fixed with new commit.